app/globals.css

```diff
@@ -84,4 +84,35 @@ body {
   border-width: 0 290px 270px 0px;
   border-color: transparent transparent rgba(255, 255, 255, 0.05) transparent;
   z-index: 1;
-}
\ No newline at end of file
+}
+.box-carousel {
+  display: flex;
+  justify-content: center;
+  margin-top: 10rem;
+}
+
+.circles {
+  width: 100%;
+}
+
+.circle {
+  border-radius: 50%;
+  width: 1.25rem;
+  height: 1.25rem;
+  cursor: pointer;
+}
+
+.carousel {
+  transition: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-timing-function: cubic-bezier(0, 0, 0.2, 1);
+}
+
+.box-depositions {
+  width: 50rem;
+}
+
+.depositions {
```

Comment:
@Keikonichi I did a refactor on this branch. I've changed Depositions and DepositoinsBox (it had a typo on Depositoins instead of Depositions), now they're called Testimonials and TestimonialsBox. Please, I didn't change CSS classes and className properties in components. Please replace everything in project to `testimonials`.

Reply:
@thiagobraga Fixed the centering of the component inside the Carousel and changed the className properties in components.